conda-envs/environment-test-py39.yml

```diff
@@ -0,0 +1,32 @@
+name: pymc3-test-py39
+channels:
+- conda-forge
+- defaults
+dependencies:
+- aesara>=2.1.0
+- arviz>=0.11.2
+- cachetools>=4.2.1
+- cloudpickle
+- fastprogress>=0.2.0
+- h5py>=2.7
+- ipython>=7.16
+- libblas=*=*mkl
+- mkl-service
+- myst-nb
+- nbsphinx>=0.4
+- numpy>=1.15.0
+- numpydoc>=0.9
+- pandas>=0.24.0
```

Comment:
This is a third different way to pin pandas.
If these differences have a good reason we should probably comment that right in the file.